comments/models.py

```diff
@@ -41,5 +42,8 @@ class Meta:
 
 
 def on_delete_comment(sender, instance, **kwargs):
+    try:
         instance.sound.post_delete_comment()
+    except ObjectDoesNotExist:
```

Comment:
`Sound.DoesNotExist`
It would be nice to have a clarifying comment here, like we do in a few other post_delete triggers. Maybe something like
"If this comment is deleted as a result of its parent sound being deleted, the sound will no longer exist so we don't need to update it"

Reply:
Comment added